website/docs/docs/cloud/git/setup-azure.md

```diff
@@ -62,22 +62,20 @@ An Entra ID admin needs to provide your new app access to Azure DevOps:
 
 ## Add another redirect URI
 
-A Microsoft Entra ID admin needs to add another redirect URI to your Entra ID application. This redirect URI will be used to authenticate the service user for headless actions in deployment environments.
+A Microsoft Entra ID admin needs to add another redirect URI to your Entra ID application. This redirect URI will be used to authenticate the service principal for headless actions in deployment environments.
```

Comment:
With service principal I don't think we need to add redirect uri to the entra app anymore. But if they are switching from service user to service principal with the same application credentials, they can just keep the redirect uri